README.md

```diff
@@ -115,6 +115,12 @@ Some of these depend on each other
           backup_password: "sample backup password"
           version: "3.2" # can be 3.0 or 3.2
         ```
+* LEMP
+* postfix
+    ```
+    postfix:
+        mailname: "{{ ansible_hostname }}"
```

Comment:
It looked like you didn't document here in the reader vars in defaults so I left it out. I can add it if you like.